src/components/Disclosures/disclosure.dev.tsx

```diff
@@ -0,0 +1,35 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
```

Comment:
Just double checkking:
Do you deliberately want to commit this story?
The story is copying what the MaterialDisclosure is doing